### DISCUSSION TITLE
Custom Screen Size?

### DISCUSSION THREAD

**Discussion by dannyaguero** [ScratchAddons/ScratchAddons#2833] [Ideas] [answered]
Can there ever be a possibility for a addon where you can choose your own custom screen size, like E羊icques?

**Comment by dannyaguero** [comment]
I know that it won't show if you post it to Scratch, with it just being cut off, but this could be a good idea for offline/HTML use!

**Reply by apple502j** [reply]
We do not have offline editors or HTML. Maybe ask TurboWarp.

**Comment by TheColaber** [comment, marked as answer]
We will not implement this because it will not be playable for those that do not have scratch add-ons. However, Turbowarp already has this feature, and you can adjust it there.
